store/tikv/region_cache.go

```diff
@@ -1272,9 +1315,10 @@ func (s *Store) reResolve(c *RegionCache) {
 		}
 	}
 	addr = store.GetAddress()
+	labels := store.GetLabels()
```

Comment:
does pd can modify exists labels in a store to another?

Reply:
Depends on what TiKV reports.

Reply:
maybe we need a new way to let tidb know labels have changed?

now, address change is triggered by KV StoreNotMatch error, how about KV's label change?

Reply:
Is there any mechanism to refresh all cached regions? If PD can't push updates to TiDBs, we need such a mechanism.